src/standard.yml

```diff
@@ -131,7 +132,8 @@ jobs:
 
       # Setup DB in this job, because it cannot be shared through the attach_workspace
       - run: bundle exec rake db:create
-      - run: bundle exec rake db:schema:load
+      - run: if [ -e db/schema.rb ]; then bundle exec rake db:schema:load; fi
+      - run: if [ -e db/structure.sql ]; then bundle exec rake db:structure:load; fi
```

Comment:
I need to fix this so that it loads the `mysql` or `postgres` library before running.

Reply:
@rdunlop let me know when you've added the fix and I'll take another look

Reply:
@giladshanan ready for re-review